app/models/bgs_dependents/base.rb

```diff
@@ -2,10 +2,51 @@
 
 module BGSDependents
   class Base < Common::Base
+    def create_person_params(proc_id, participant_id, payload)
+      {
+        vnp_proc_id: proc_id,
+        vnp_ptcpnt_id: participant_id,
+        first_nm: payload['first'],
+        middle_nm: payload['middle'],
+        last_nm: payload['last'],
+        suffix_nm: payload['suffix'],
+        brthdy_dt: format_date(payload['birth_date']),
+        birth_state_cd: payload['place_of_birth_state'],
+        birth_city_nm: payload['place_of_birth_city'],
+        file_nbr: payload['va_file_number'],
+        ssn_nbr: payload['ssn'],
+        death_dt: format_date(payload['death_date']),
+        ever_maried_ind: payload['ever_married_ind'],
+        vet_ind: payload['vet_ind'],
+        martl_status_type_cd: 'Married'
```

Comment:
I assume this hard-coding is intentional?

Reply:
Good catch.  This has been updated with some logic to populate "Never Married", "Married", or "Separated"